### PR TITLE
Awesome better README

### PR DESCRIPTION
**Pull request by bobiechen-twilio**
Improve the README with instructions of how to get the quickstart application up and running.

### CODE REVIEW

**Comment by kbagchiGWC** [comment]
👍

**Review by paynerc — COMMENTED** [review]
In the base url png... Do you want to show that url since it points to something? Or do you just want to show the line where you change it with the placeholder text showing?

**Comment by bobiechen-twilio** [comment]
The url in the base-url image is actually the same from the url we have in the create-TwiML-app png, so just tried to make them look consistent :p

File: README.md [hunk 1]
```diff
@@ -1,14 +1,124 @@
...
+## More Documentation
+You can find more documentation on getting started as well as our latest AppleDoc below:
+
+* [Getting Started](https://www.twilio.com/docs/api/voice-sdk/ios/getting-started)
+* [AppleDoc](https://media.twiliocdn.com/sdk/ios/voice/latest/docs)
+
+## Issues and Support
+Please file any issues you find here on Github [ObjC](https://github.com/twilio/voice-quickstart-objc) [Swift](https://github.com/twilio/voice-quickstart-swift).
+
+For general inquiries related to the Voice SDK you can file a support ticket.
```
Comment (paynerc) [inline README.md]: Should we include a link to filing a support ticket?

**Review by paynerc — APPROVED** [review]
I think this looks pretty good.

👍

File: README.md (hunk 1 above)
Comment (bobiechen-twilio) [reply README.md]: Yes, we should. Will update.

**Review by paynerc — APPROVED** [review]
LGTM

**Pull request merged**